docs/siem-apis.asciidoc

```diff
@@ -2,6 +2,8 @@
 [[security-apis]]
 = Elastic Security APIs
 
+WARNING: Any request or response examples created in {kib} Dev Tool Console cannot be used to create an API request.
```

Comment:
@DonNateR I don't think this is what we're trying to say. You _can_ make API requests via the dev tool console, you just can't copy and paste the examples; you have to type them in manually, I think. @louisong @spong is this correct?

Reply:
My impression from reading the issue is that the call itself wouldn't work, even if the responses in the Dev Console were valid.

Reply:
Gotcha - thanks. Slight suggestion.

```suggestion
WARNING: Do not use the {kib} Dev Tool Console to make any {es-sec} API requests. 
```

Reply:
Yeah specifically here, Kibana API requests cannot be made via the dev tools. This is the message used on the kibana docs here: https://www.elastic.co/guide/en/kibana/7.10/using-api.html. Perhaps we should just use that message as well?

![image](https://user-images.githubusercontent.com/2946766/106971343-fa812180-670b-11eb-8a0e-3baa88a396b2.png)